src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Components/ComponentDesignTimeNodeWriter.cs

```diff
@@ -452,7 +452,6 @@ public override void WriteComponent(CodeRenderingContext context, ComponentInter
             if (node.Component.SuppliesCascadingGenericParameters())
             {
                 typeInferenceCaptureScope = context.CodeWriter.BuildScope();
-                context.CodeWriter.Write("global::");
```

Comment:
Why are we removing these?

Reply:
Discussed in https://github.com/dotnet/razor/pull/9385#discussion_r1353359545